From 5148e0a7e3cd4293416b969f2c563faa6aca27b2 Mon Sep 17 00:00:00 2001
From: Joe Slater <jslater@windriver.com>
Date: Mon, 11 Mar 2013 15:47:54 -0700
Subject: [PATCH] pacemaker: do not use libgnutls-config

Do not try to use libgnutls-config when configuring.
It has been deprecated and we do not supply it.  If a
host version is found, bad things can happen.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Joe Slater <jslater@windriver.com>
Signed-off-by: Jackie Huang <jackie.huang@windriver.com>

---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index e930e33..2eade3c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1707,9 +1707,9 @@ dnl ========================================================================
 AC_CHECK_HEADERS(gnutls/gnutls.h)
 AC_CHECK_HEADERS(security/pam_appl.h pam/pam_appl.h)
 
-dnl GNUTLS library: Attempt to determine by 'libgnutls-config' program.
-dnl If no 'libgnutls-config', try traditional autoconf means.
-AC_PATH_PROGS(LIBGNUTLS_CONFIG, libgnutls-config)
+dnl libgnutls-config has been deprecated, so we use pkg-config
+
+LIBGNUTLS_CONFIG="pkg-config gnutls"
 
 if test -n "$LIBGNUTLS_CONFIG"; then
 	AC_MSG_CHECKING(for gnutls header flags)
